projects/demo/src/pages/recipes/card/examples/1-basic/component.ts

```diff
@@ -20,7 +20,7 @@ export class CardDocExample1 {
             ' ',
             ...Array(4).fill(/\d/),
             ' ',
-            ...Array(3).fill(/\d/),
```

Comment:
Hi! 👋  I did not find information that a payment card can contain more than 19 digits. https://en.wikipedia.org/wiki/Payment_card_number
Am I wrong?